time/src/main/proto/spine/time_options.proto

```diff
@@ -53,46 +53,45 @@ extend google.protobuf.FieldOptions {
     TimeOption when = 73819;
 }
 
-// The field value is point in time which lies in the future or in the past.
+// Specifies that the field value is a point in time lying either in the future or in the past.
+//
+// Applicable only to `google.protobuf.Timestamp` and types marked as `io.spine.time.Temporal`:
```

Comment:
`Temporal` is a Java interface. Ideally, it should not be mentioned on the level of Proto model at all because the model should be separated from the target languages. But I decided at least to provide a list  of Proto messages that have this interface.

This is the same same enumeration of supported types I'm going to print in a compilation error reported when an unsupported field type is encountered.

Reply:
> * give the reference to `time` library and the corresponding package instead of enumerating the types here.

We are in the Spine Time library. Can we just reference the package `spine.time`, assuming we intentionally don't have proto packages for options?